src/v/cluster/feature_manager.cc

```diff
@@ -299,20 +299,18 @@ ss::future<> feature_manager::maybe_log_license_check_info() {
         // Shutting down - next iteration will drop out
```

Comment:
> redpanda/admin/server, cluster/feature_manager: do not check feature:…
…:license
> as it is on everywhere now

Commit message subject should not wrap. Body should stand on its own.

Reply:
oops, sorry, I thought the limit was 80, but it's 50 for header and 72 for body, will fix